all-spark-notebook/test/test_spark_notebooks.py

```diff
@@ -7,16 +7,17 @@
 import os
 
 LOGGER = logging.getLogger(__name__)
+THIS_DIR = os.path.dirname(os.path.realpath(__file__))
 
 
 @pytest.mark.parametrize(
     "test_file",
     # TODO: add local_sparklyr
-    ["local_pyspark", "local_spylon", "local_sparkR", "issue_1168"],
+    ["local_pyspark", "local_spylon", "local_sparkR", "issue_1168"]
```

Comment:
Do we have an auto formatter that opts to not using trailing commas?

Reply:
The `black` autoformatter enforces trailing commas.

I've come to appreciate that as you can remove and add a line without seeing a git diff on multiple lines because a comma would need to be updated in a separate line than the line you added or removed.

Reply:
> Do we have an auto formatter that opts to not using trailing commas?

No, we don't, right now, as you already noticed.